scheduler/src/cook/mesos/scheduler.clj

```diff
@@ -378,16 +391,31 @@
    the offer.
 
    Returns a list of tasks that got matched to the offer"
-  [schedule {{:keys [cpus mem]} :resources :as offer}]
-  (log/debug "Matching offer " offer " to the schedule" schedule)
-  (loop [[candidates & remaining-prefixes] (prefixes schedule)
-         resources-so-far {:mem 0 :cpus 0}
-         prev []]
-    (let [required (merge-with + resources-so-far
-                               (select-keys (util/job-ent->resources (last candidates)) [:mem :cpus]))]
-      (if (and candidates (>= cpus (:cpus required)) (>= mem (:mem required)))
-        (recur remaining-prefixes required candidates)
-        prev))))
+  [^TaskScheduler fenzo considerable offers db fid]
+  (log/debug "Matching" (count offers) "offers to" (count considerable) "jobs with fenzo")
+  (let [t (System/currentTimeMillis)
+        leases (mapv #(->VirtualMachineLeaseAdapter % t) offers)
+        requests (mapv (fn [job]
+                         (let [job-id (:db/id job)]
```

Comment:
Nowhere. It looks like leftover cruft, there was probably a log statement including it at some point.